frontend/src/store/cloudProfile/helper.js

```diff
@@ -74,12 +74,14 @@ export function findVendorHint (vendorHints, vendorName) {
 export function decorateClassificationObject (obj) {
   const classification = obj.classification ?? 'supported'
   const isExpired = obj.expirationDate && moment().isAfter(obj.expirationDate)
+
   return {
     ...obj,
     isPreview: classification === 'preview',
     isSupported: classification === 'supported' && !isExpired,
     isDeprecated: classification === 'deprecated',
     isExpired,
+    isExpirationWarning: !!obj.expirationDate && moment(obj.expirationDate).diff(moment(), 'd') < 30,
```

Comment:
Same as below. The `&&` makes the expression return a boolean and we do not need `!!` to enforce boolean here.

Reply:
see comment above

Reply:
see also https://262.ecma-international.org/5.1/#sec-11.11